vnext/Microsoft.ReactNative/Fabric/platform/react/renderer/graphics/PlatformColorUtils.cpp

```diff
@@ -142,4 +144,35 @@ winrt::Windows::UI::Color ResolvePlatformColor(const std::vector<std::string> &s
   return {};
 }
 
+winrt::Windows::UI::Color GetTextInputPlaceholderColor(
+    bool isFocused,
+    const winrt::Windows::UI::Color &backgroundColor) {
+  // In high contrast mode, always use system GrayText for accessibility
+  HIGHCONTRAST hcInfo = {};
+  if (SystemParametersInfo(SPI_GETHIGHCONTRAST, 0, &hcInfo, 0) && (hcInfo.dwFlags & HCF_HIGHCONTRASTON)) {
```

Comment:
I'd be shocked if this was the only "is high contrast" check that ends up persisting in PlatformColor. Worth making into a clear helper method and a clear legible boolean result.